src/docs/product/releases/usage/archive-release.mdx

```diff
@@ -4,15 +4,15 @@ sidebar_order: 30
 description: "Learn how to permanently hide or archive a release."
 ---
 
-Since an event received from a deleted release automatically recreates it, Sentry has an option to hide releases permanently. In addition, archiving releases can be useful if you're no longer interested in viewing the release or if you've accidentally created a release you no longer want to track.
+Since an event received from a deleted release automatically recreates it, Sentry has an option to hide, or archive, releases. Archiving can also be useful if you're no longer interested in viewing a release or if you've accidentally created a release you no longer want to track.
 
 ## Using Sentry.io
 
-Archiving in Sentry.io removes the release from the list of releases you can select to view.
+Archiving in [sentry.io](https://sentry.io) removes the release from the list of releases you can select to view:
 
 ![archive release ui](release-archive.png)
 
-Archived releases won't show up in Releases view unless you filter for them:
+Archived releases won't show up in **Releases** unless you filter for them:
```

Comment:
"unless you filter for them" feels a little weird. They're filtered by default and you need to change the filters show archived

![image](https://user-images.githubusercontent.com/1400464/168880160-de9e69c6-8440-46a3-8309-1a62a619b865.png)